FHEM/14_SD_WS_Maverick.pm

```diff
@@ -62,36 +63,6 @@ SD_WS_Maverick_Define($$)
   $modules{SD_WS_Maverick}{defptr}{$a[2]} = $hash;
   $hash->{STATE} = "Defined";
   
-  my $name= $hash->{NAME};
```

Comment:
Kannst Du erläutern, wieso dieser Code entfernt wird.

Ich finde einfach keinen Zusammenhang mit "TFA 14.1504"

Reply:
Die lokale Variable `$name` wurde ausschließlich in dem gelöschten Codeblock verwendet und ist deshalb jetzt überflüssig. Da sie innerhalb der Subroutine `SD_WS_Maverick_Define` definiert wird, ist sie nirgendwo anders sichtbar.

Reply:
Ich habe mich nicht gut ausgedrückt.
Ich habe mich gewundert, wieso Zeile 65 - 94 gelöscht werden und wieso das für das TFA 14.1504 Gerät notwendig ist.

Reply:
Ah, OK. Der gelöschte Block enthält ausschließlich die Definition von `stateFormat`, wo der State des Gerätes gesetzt wird. Diesen Code habe ich nach `SD_WS_Maverick_UpdateState` übernommen (und etwas refaktorisiert). Deshalb ist `stateFormat` jetzt frei geworden für den Anwender und braucht hier nicht mehr angelegt zu werden.

Diese Änderung ist streng genommen nicht für die Unterstützung des _TFA 15.1504_ notwendig, ist aber in der Diskussion des zugehörigen Issues 167 [thematisiert](https://github.com/RFD-FHEM/RFFHEM/issues/167#issuecomment-526241990) worden.

Reply:
Okay, hoffen wir mal, dass es nicht zur Verwunderung kommt, dass stateformat beim Initialisieren nicht mehr vorbelegt wird.